lib/Mojolicious/Guides/Growing.pod

```diff
@@ -245,6 +244,8 @@ concerns more visible.
 
   use strict;
   use warnings;
+  use feature qw(signatures);
```

Comment:
This can be one line, as experimental is core in all versions that have the signatures feature: `use experimental qw(signatures);`